tutorials/detecting-changes-in-sentinel-1-imagery-pt-4/Tutorial_pt_4.ipynb

```diff
@@ -0,0 +1,400 @@
+{
```

Comment:
<ul><li>Please also use the "&gt;" quote indicator for the two italicized quotes</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/422/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Done.</p>